requirements.txt

```diff
@@ -1,4 +1,5 @@
 docker-compose==1.4.2
+docopt==0.6.2
```

Comment:
Noted. Will avoid this in future patches. Small changes that I've forgot to commit in their proper patch will go directly to the mainline branch.